providers/capvcd/standard/test_data/cluster_values.yaml

```diff
@@ -1,11 +1,8 @@
 baseDomain: test.gigantic.io
 controlPlane:
-  catalog: giantswarm
   replicas: 3
+  diskSizeGB: 30
```

Comment:
How come this is needed for standard but not upgrade suite?

Reply:
It was an oversight on my part, thanks.
Fixed

Reply:
😕  I still see it on the PR. Is it just me?

Reply:
Oh, you need it on both! Got it!